src/serialization/Portable.ts

```diff
@@ -84,8 +91,13 @@ export interface PortableWriter {
 
     writeShortArray(fieldName: string, shorts: number[]): void;
 
+    /** @deprecated since version 4.1.
```

Comment:
There are some leftover `4.1`s in this file

Reply:
Good catch 👍🏻 I went over all files but looks like missed these.